explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -44,39 +45,43 @@ object ExploreLayout {
     AppCtx.using { implicit ctx =>
       HelpCtx.usingView { helpCtx =>
         val helpView = helpCtx.zoom(HelpContext.displayedHelp)
-        SidebarPushable(
-          Sidebar(
-            width = SidebarWidth.Wide,
-            direction = SidebarDirection.Right,
-            animation = SidebarAnimation.Overlay,
-            visible = helpView.get.isDefined
-          )(
-            helpView.get
-              .map { h =>
-                // Lazy load the React component for help
-                val prom = js.dynamicImport {
-                  new HelpLoader().loadHelp(helpCtx.get, h)
+        GlobalHotKeys(keyMap = KeyMap("CLOSE_HELP" -> "ESC"),
```

Comment:
We should probably activate this only when the help is open, so that we can use the ESC key for other purposes in other parts of the application.

I think this can be achieved by just moving this `GlobalHotKeys` to the help component.

Also, it is recommended to use `GlobalHotKeys` as a stand alone component (the exception is when there's another `GlobalHotKeys` further down the tree which should override this one): https://github.com/greena13/react-hotkeys#globalhotkeys-component

Reply:
Thanks, I implemented it as you suggested and it seems a lot better